src/pages/posts/components/PostForm.js

```diff
@@ -138,7 +140,7 @@ function PostForm({ onCreate, data = {} }) {
                   <Option key={'draft'} value={'draft'}>
                     Draft
                   </Option>
-                  <Option key={'publish'} value={'publish'}>
+                  <Option disabled={!(actions.includes('admin') || actions.includes('publish'))} key={'publish'} value={'publish'}>
```

Comment:
Don't show the option instead of disabling it.

Reply:
ok